src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepositoryTest.java

```diff
@@ -331,15 +331,16 @@ void should_return_envelopes_count_for_container_and_time_range() {
     @Test
     void should_return_envelopes_for_the_requested_date() {
         //given
-        addEnvelope("C1", "f1");
-        addEnvelope("C3", "f2");
-        addEnvelope("C3", "f3");
+        UUID id1 = addEnvelope("f1", "C1");
+        UUID id2 = addEnvelope("f2", "C2");
+        UUID id3 = addEnvelope("f3", "C2");
```

Comment:
Not sure how stable this test will be if we save envelopes one after another. Can they be saved within the same millisecond?

Reply:
I'll add `Thread.sleep()` with 2 milliseconds